platform/security/common/src/main/java/org/codice/ddf/security/common/jaxrs/RestSecurity.java

```diff
@@ -46,13 +42,14 @@
  */
 public final class RestSecurity {
 
-    // SAML_COOKIE_NAME is not available in SecurityConstants 2.4.0
-    public static final String SECURITY_COOKIE_NAME = "org.codice.websso.saml.token";
```

Comment:
@stustison Won't completely removing support for the old cookie make this DDF incompatible with older versions of DDF REST security?  If so, we should probably document that.